'es6-promise/auto' not working (4.0.0)

Looks like `auto.js` wasn't published to NPM:

`Error: Cannot find module 'es6-promise/auto'`

Upon fixing that manually to confirm, I also get (on Node < 4):

`Error: Cannot find module '.'`

--- a/auto.js
+++ b/auto.js
@@ -1,4 +1,4 @@
 // This file can be required in Browserify and Node.js for automatic polyfill
 // To use it:  require('es6-promise/auto');
 'use strict';
-module.exports = require('.').polyfill();
+module.exports = require('./').polyfill();

--- a/package.json
+++ b/package.json
@@ -11,6 +11,7 @@
   "files": [
     "dist",
     "lib",
+    "auto.js",
     "es6-promise.d.ts",
     "!dist/test"
   ],

